client/src/utils/ajax.ts

```diff
@@ -230,6 +231,13 @@ export function postWebhookToken(): Promise<void> {
   return post('/user/webhook-token', {});
 }
 
+export function postSaveChallenge(body: {
+  id: string;
+  challengeFiles: ChallengeFiles;
```

Comment:
```suggestion
  files: ChallengeFiles;
```
The type isn't quite right, since ChallengeFiles includes fileKey, but the property is right.